src/test/java/nl/cherement/jak/service/ColumnServiceTests.java

```diff
@@ -31,23 +28,16 @@
 
     @BeforeEach
     public void initialize() {
-        column = new ColumnEntity();
-        column2 = new ColumnEntity();
-        columns = new ArrayList<ColumnEntity>();
-
-        column.setId(1l);
-        column2.setId(1l);
+        ColumnEntity column = new ColumnEntity();
```

Comment:
de naam en de declaratie is hier op 1 line maar overal is het opgesplitst in 2, zelfs met columns in hetzelfde file.

Reply:
zelfde als bij de andere. Heb er een local variable van gemaakt omdat hij alleen in deze methode gebruikt wordt.